.travis.yml

```diff
@@ -11,3 +11,4 @@ env:
     - JAVA_OPTS="-XX:MaxPermSize=2g -Xmx4g"
 script:
   - mvn clean install -Pqulice --errors
+  - mvn clean install -DargLine="-Duser.language=fr -Duser.country=FR"
```

Comment:
@mkordas travis build is not the right place for testing, please create invoker test (there is a bunch of them in the qulice-maven-plugin module)